autoload/neomake/makers/ft/haskell.vim

```diff
@@ -1,10 +1,23 @@
 
 function! neomake#makers#ft#haskell#EnabledMakers()
-    return ['ghcmod', 'hdevtools', 'hlint']
+    let makers = []
+    if executable('ghc-mod')   | call add(makers, 'ghcmod')    | endif
+    if executable('hdevtools') | call add(makers, 'hdevtools') | endif
+    if executable('hlint')     | call add(makers, 'hlint')     | endif
+    if executable('liquid')    | call add(makers, 'liquid')    | endif
+    return makers
```

Comment:
Hmm, well, both ````stack```` and the executable, e.g. ````hlint```` are needed for running the ````hlint```` command via ````stack````. So, my idea was to check for both. But, in fact, one could have an executable that *only* works when run via ````stack````, because ````stack```` adds search paths to look for the executable in, i.e. the current Haskell project's binary directory. So...a better solution would be to run a test like ````stack exec hlint -V```` to see if stack is installed AND whether it can find ````hlint````. I'll have to think about this a bit.